aarch64_extension.cpp

```diff
@@ -492,7 +623,7 @@ class AArch64ArchitectureExtension : public ArchitectureHook {
   bool GetInstructionLowLevelIL(const uint8_t* data, uint64_t addr, size_t& len,
                                 LowLevelILFunction& il) override {
     cs_insn* instr;
-    size_t count = cs_disasm(disassembler.Get(), data, len, addr, 0, &instr);
+    size_t count = cs_disasm(disassembler.Get(), data, 4, addr, 0, &instr);
```

Comment:
Why the change to `4`?

Reply:
We may want to `#define` that, but we're called for every instruction - we cannot return multiple instructions in one call, so it's a waste to have capstone disassemble the rest.

Reply:
`#define` sounds good.